app/cli/internal/policydevel/eval.go

```diff
@@ -28,6 +28,10 @@ import (
 	"github.com/chainloop-dev/chainloop/pkg/attestation/crafter/materials"
 )
 
+const (
+	DefaultMaterialName = "material"
```

Comment:
do you need to expose this constant? 

```suggestion
	defaultMaterialName = "material"
```